frontend/translations.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
   SendAddressInvalidAddress: () => 'Invalid address',
   SendAmountIsNan: () => 'Invalid format: Amount has to be a number',
-  SenAmountIsNotPositive: () => 'Invalid format: Amount has to be a positive number',
-  SendAmountInsufficientFunds: ({balance}) => `Insufficient funds. Your balance is ${balance} ADA.`,
-  SendAmountInsufficientFundsForFee: ({balance, transactionFee}) => `Insufficient funds to cover the transaction fee. 
-    Cannot send more than ${balance - transactionFee} ADA.`,
+  SendAmountIsNotPositive: () => 'Invalid format: Amount has to be a positive number',
+  SendAmountInsufficientFunds: ({balance}) =>
+    `Insufficient funds for the transaction. Your balance is ${balance / 1000000} ADA.`,
```

Comment:
Could the division lead to some weird values? A true solution should be (assuming positive balance)
`${Math.floor(balance/1000000)} . ${figure_out_how_to_pad_and_strip_zeroes(balance % 100000)}`
but I guess for now a TODO comment would do